CI Error: selectNoneIsVisibleOnceSelectAllCalled

https://travis-ci.org/github/ankidroid/Anki-Android/jobs/735935870

```
com.ichi2.anki.CardBrowserTest > selectNoneIsVisibleOnceSelectAllCalled FAILED
    java.lang.AssertionError: 
    Expected: is <true>
         but: was <false>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
        at com.ichi2.anki.CardBrowserTest.selectNoneIsVisibleOnceSelectAllCalled(CardBrowserTest.java:88)
```

--- a/AnkiDroid/src/test/java/com/ichi2/anki/CardBrowserTest.java
+++ b/AnkiDroid/src/test/java/com/ichi2/anki/CardBrowserTest.java
@@ -75,7 +75,7 @@ public class CardBrowserTest extends RobolectricTest {
     public void selectAllIsNotVisibleOnceCalled() {
         CardBrowser browser = getBrowserWithMultipleNotes();
         selectMenuItem(browser, R.id.action_select_all);
-        advanceRobolectricLooper();
+        advanceRobolectricLooperWithSleep();
         assertThat(browser.isShowingSelectAll(), is(false));
     }
 
@@ -83,7 +83,7 @@ public class CardBrowserTest extends RobolectricTest {
     public void selectNoneIsVisibleOnceSelectAllCalled() {
         CardBrowser browser = getBrowserWithMultipleNotes();
         selectMenuItem(browser, R.id.action_select_all);
-        advanceRobolectricLooper();
+        advanceRobolectricLooperWithSleep();
         assertThat(browser.isShowingSelectNone(), is(true));
     }
 

